### PR TITLE
completed +help comments for places they were missing

### PR DESCRIPTION
**Pull request by baudtack**

### CODE REVIEW

**Comment by baudtack** [comment]
@philipcmonk can you review the comments on spider and ph generators here? I'm not sure I understood them all.

File: pkg/arvo/gen/aqua/pill.hoon [hunk 1]
```diff
@@ -1,3 +1,4 @@
+::  Produce a pill using aqua
```
Comment (philipcmonk) [inline pkg/arvo/gen/aqua/pill.hoon]:
```suggestion
::  Produce a pill for aqua
```

File: pkg/arvo/gen/azimuth-tracker/watch.hoon [hunk 2]
```diff
@@ -1,3 +1,4 @@
+::  Add a new url to the watch list
```
Comment (philipcmonk) [inline pkg/arvo/gen/azimuth-tracker/watch.hoon]:
```suggestion
::  Change node url for azimuth-tracker
```

File: pkg/arvo/gen/ph/cancel.hoon [hunk 3]
```diff
@@ -1,3 +1,4 @@
+::  Cancel a ph command
```
Comment (philipcmonk) [inline pkg/arvo/gen/ph/cancel.hoon]: The entire gen/ph directory should be deleted now that app/ph no longer exists (it's a library now and doesn't use generators)

**Comment by ixv** [comment]
@rmariani ?

**Comment by ixv** [comment]
@baudtack This has been sitting here for a while, please advise. Should I wait for review from @rmariani or just merge it now?

**Comment by rmariani** [comment]
Slipped my mind, let me review now

**Comment by ixv** [comment]
Thanks

**Pull request merged**